src/glmfit.jl

```diff
@@ -115,7 +115,23 @@ function confint(obj::AbstractGLM, level::Real)
 end
 confint(obj::AbstractGLM) = confint(obj, 0.95)
 
-deviance(m::AbstractGLM)  = deviance(m.rr)
+deviance(m::AbstractGLM) = deviance(m.rr)
+
+function loglikelihood(m::AbstractGLM)
```

Comment:
@dmbates Makes sense. For my education, do you know how to compute the log-likelihood from the deviance and the null deviance in general? To my surprise, I could find no explicit reference about this.

@andreasnoack The point is, the log-likelihood for GLMs is quite a well-defined measure, it would be terribly confusing to return something different from other software. But as Douglas said, it's not needed that often (e.g. the R² is currently computed from the deviance).